plugins/rapid7_insightidr/unit_test/util.py

```diff
@@ -0,0 +1,93 @@
+import json
+import logging
+import os
+
+from komand.exceptions import PluginException
+from komand_rapid7_insightidr.connection import Connection
+from komand_rapid7_insightidr.connection.schema import Input
+from requests.models import HTTPError
+
+
+class Meta:
+    version = "0.0.0"
+
+
+class Util:
+    @staticmethod
+    def default_connector(action, connect_params: object = None):
+        default_connection = Connection()
+        default_connection.meta = Meta()
+        default_connection.logger = logging.getLogger("connection logger")
+        if connect_params:
+            params = connect_params
+        else:
+            params = {
+                Input.URL: "https://rapid7.com",
+                Input.API_KEY: {"secretKey": "4472f2g7-991z-4w70-li11-7552w8qm0266"},
```

Comment:
This is a fake API key right?

Reply:
Yes :)

Reply:
Okay, just making sure! We can never be too careful!